### DISCUSSION TITLE
[2358720]黑神话：悟空 {仅本体}

### DISCUSSION THREAD

**Discussion by WilliamK7** [pjy612/SteamManifestCache#773] [Share]
### Steam 游戏ID

2358720

### 密钥列表

```hash
2358721;78d5a284965b2206de191dc4ca99d43c9caf5b5e78b47cf0c366fbbb884190d8
```

### 清单附件

[2358721_7558476378129500741.zip](https://github.com/user-attachments/files/16683855/2358721_7558476378129500741.zip)

**Comment by buaa1205** [comment]
哭哭，已经限制了，会报错

**Reply by WilliamK7** [reply]
这种有 D加密 的游戏都是需要用正版账号启动一次才能玩的，报错不是资源本身的问题

**Reply by WilliamK7** [reply]
Sorry, but I'm using a private account, so I'm afraid that I can't share it with you.

**Reply by buaa1205** [reply]
那是不是买退就可以了

**Reply by WilliamK7** [reply]
对，但是挺麻烦的，听说不能更新游戏，不能更新显卡驱动，不能更新系统

**Reply by buaa1205** [reply]
显卡驱动都不行。。。这也太难了

**Reply by itzzzme** [reply]
the game is not getting added after I put the appid in the steamtools can you tell me what am I doing wrong, I also created the lua file and put the manifest file into depotcache but it's still not adding the game into my account

**Reply by buaa1205** [reply]
1.Drag the url of the game to steamtools, you will add the game
2.copy manifest to steam document and modify config.vdf like
"depots"
{
	"1716741"
	{
		"DecryptionKey" "14c0bdb25d858d05a7381ba67a373aa99b50c5bd4cf78f1895751d418de29ceb"
	}
You will download game
But D encryption will cause 88500000error if you cant run game once

**Reply by itzzzme** [reply]
@buaa1205 I have done everything it's downloading 
![Screenshot 2024-08-24 105142](https://github.com/user-attachments/assets/1bde66ef-735e-4e5e-895c-2f86c5d7e939)
but will I get the denuvo error?

**Reply by WilliamK7** [reply]
> will I get the denuvo error?

As long as you don't buy the game, there definitely will be a Denuvo error.

**Reply by itzzzme** [reply]
@WilliamK7

> This kind of D-encrypted game needs to be started once with a genuine account before it can be played. The error is not a problem with the resource itself.

So if you run it once then can we play the game is that what you want to say?

**Reply by WilliamK7** [reply]
You need to have a genuine account and login on your computer, then run the game once. After that, you can turn on the offline mode in steam and play.

But this method is quite unstable

**Reply by itzzzme** [reply]
if possible can I get the account for max 1 day??

**Reply by buaa1205** [reply]
You can get a refund after you buy it

**Reply by WilliamK7** [reply]
Sorry, but actually I myself haven't bought the game either. The key and manifest file were collected on the Internet.

**Reply by itzzzme** [reply]
@WilliamK7 so there is no solution??

**Reply by WilliamK7** [reply]
I'm afraid that there is no solution as far as I know.😥

**Reply by itzzzme** [reply]
@WilliamK7 https://www.digitalgamezone.co.in/product/31274326/Black-Myth-Wukong--Digital-Deluxe-Edition--PC-?vid=5867819

you can try this solution. I don't know which country you are from but this seems pretty cheap

**Reply by itzzzme** [reply]
> I'm afraid that there is no solution as far as I know.😥

@WilliamK7 so the key and manifest file are just useless right?

**Reply by itzzzme** [reply]
@WilliamK7   btw can you tell me from where you found the lua and manifest

**Reply by WilliamK7** [reply]
> from where you found the lua and manifest

https://lixianla.com/

There were some genuine accounts shared on this site several days ago, but they now all require mail authentication

**Reply by itzzzme** [reply]
> > from where you found the lua and manifest
> 
> https://lixianla.com/
> 
> There were some genuine accounts shared on this site several days ago, but they now all require mail authentication

@WilliamK7  bad luck then  :)

**Reply by buaa1205** [reply]
> > from where you found the lua and manifest
> 
> https://lixianla.com/
> 
> There were some genuine accounts shared on this site several days ago, but they now all require mail authentication

我看你发的网站里已经有新技术了，可以跳过steam玩。

**Reply by itzzzme** [reply]
> > > from where you found the lua and manifest
> > 
> > 
> > https://lixianla.com/
> > There were some genuine accounts shared on this site several days ago, but they now all require mail authentication
> 
> I see that the website you posted already has new technology, so you can skip steam and play.

@buaa1205 how to do this can you share the link

**Reply by WilliamK7** [reply]
> 我看你发的网站里已经有新技术了，可以跳过steam玩。

真的假的

**Reply by pjy612** [reply]
我感觉你们聊这么多 不如去 steamtools 的 tg 里面。。。他们都玩了好几天了。。。

**Reply by WilliamK7** [reply]
哇，刚刚才看到大佬也在 steamtools 的论坛里。之前 steamtools 不是说不提供黑神话的解锁嘛，我就只收集了清单和密钥，没去试

**Reply by itzzzme** [reply]
@pjy612 shit!!! is it for real???!!!!

**Reply by itzzzme** [reply]
@WilliamK7 
![image](https://github.com/user-attachments/assets/ffa6ee55-812e-4fe1-ac3c-fde07f8e7dcd)
can anyone tell me why the full game size is not showing I have done everything in correct order

**Reply by itzzzme** [reply]
@WilliamK7 do you know how to remove any game from the list and again add it to the library

**Reply by WilliamK7** [reply]
From which list?

**Reply by itzzzme** [reply]
> From which list?

@WilliamK7 from steam library list

**Reply by WilliamK7** [reply]
> I have done everything in correct order

If you did it in this order :
> 1.Drag the url of the game to steamtools, you will add the game
2.copy manifest to steam document and modify config.vdf like
"depots"
{
"1716741"
{
"DecryptionKey" "14c0bdb25d858d05a7381ba67a373aa99b50c5bd4cf78f1895751d418de29ceb"
}
You will download game

Then for step 1, drag the url again to steamtools while holding the Ctrl key; for step 2, just manually delete what you added

**Reply by itzzzme** [reply]
@WilliamK7 problem is solved

![Screenshot 2024-08-24 105142](https://github.com/user-attachments/assets/0828faf2-c106-4856-97d6-733e5e2a86cc)

tysm!!

**Comment by itzzzme** [comment]
![image](https://github.com/user-attachments/assets/a01669c0-a512-4d4f-8cc4-8616207fc170)
@WilliamK7 @pjy612 it won't work it gives the d-encryption error

**Reply by pjy612** [reply]
Denuvo didn't crack it, only bypassing it with the ownership of other accounts. See tutorials on the steamtools forum or Telegram https://t.me/steamtool for details

**Reply by itzzzme** [reply]
> Denuvo didn't crack it, only bypassing it with the ownership of other accounts. See tutorials on the steamtools forum or Telegram https://t.me/steamtool for details

@pjy612 can you kindly send the link of the post which describes the  method to  bypass the encryption and fix the 8850000 error. I tried to join the telegram group it didn't let me join the group it's saying that "Group is not accessible"

**Reply by lgjx123** [reply]
> Denuvo didn't crack it, only bypassing it with the ownership of other accounts. See tutorials on the steamtools forum or Telegram https://t.me/steamtool for details

一样，显示群组为私密无法加入群组。是需要反向主动拉还是什么？

**Comment by XiaoAQWQ** [comment]
好像豪华版dlc直接把id拖进去就行了，今天steamtools解除了对黑猴的限制

**Comment by DaiZiSan** [comment]
为什么我的下载黑神话悟空会显示内容配置文件不可用有那个大佬知道吗

**Reply by itzzzme** [reply]
> Why does my download of Black Myth Wukong show that the content configuration file is unavailable? Does anyone know?

@DaiZiSan  send the ss

**Reply by DaiZiSan** [reply]
?

**Reply by itzzzme** [reply]
> ?

screenshot

**Reply by DaiZiSan** [reply]
![截屏2024-08-26 14 30 31](https://github.com/user-attachments/assets/a654346a-ceb4-4fd1-9a92-c1fea40bd2e0)

**Reply by itzzzme** [reply]
1.Drag the url/appid(i.e:2358720 for black myth wukong) of the game to steamtools, you will add the game
2.copy manifest to steam document and modify config.vdf like
"depots"
{
"2358721"
{
"DecryptionKey" "78d5a284965b2206de191dc4ca99d43c9caf5b5e78b47cf0c366fbbb884190d8"
}
3. Download the game

**Comment by FaizGhazali** [comment]
i wonder how to use this apps?
do i need to buy it? i already setup github token and also steam directory in the config

However, when i paste apps id into this app, it just crash

![image](https://github.com/user-attachments/assets/aa3a5115-36e5-4279-a2b0-9c41c6db1075)
